### ISSUE TITLE
What's Envy all about?

### ISSUE THREAD

**Issue by steellg2ld** [Unarray/cli-envy#1]
![omar-sy](https://github.com/user-attachments/assets/ff5b878b-0401-4c34-9103-3c174e92ae62)

**Comment by Unarray** [comment]
The application was created above all to simplify my personal use. This CLI allows me to centralize, manage and use my environment variables efficiently.

On some projects, I have to juggle with several environment files (local, dev, PR, stage, prod, etc.). Thanks to this application, I can define a single folder to group all these resources together. For example:

```
📂my-resources-dr/
├── 📂project-1/
│ ├── 📄prod
│ ├── 📄stage
│ ├── 📄dev
│ └── 📄local
├── 📂project-2/
│ ├── 📄prod
│ ├── 📄dev
│ └── 📄local
└── 📄local
```

When I'm working on `project-1`, I can switch from a `dev` environment to `local` in a single command, like `envy get project-1/local > .env`.

The idea came from my use of [`git worktree`](https://git-scm.com/docs/git-worktree), which facilitates simultaneous work on multiple branches without the need for [`git stash`](https://git-scm.com/docs/git-stash/fr) or intermediate commits. Thanks to this approach, I no longer have to search and copy/paste my environment variables every time I create a new branch. If I modify the variables, I can easily retrieve them on all my branches via `envy` or `envy get project-1/local > .env`.

In short, this solution is a simplified local alternative to tools such as [Infisical](https://infisical.com/) or [dotenv](https://www.dotenv.org/). It allows you to manage not only environment variables, but also any text file!
